classroom.rb

```diff
@@ -14,24 +14,36 @@ def add_student
 end
 
 class Book
+  @@books = []
```

Comment:
- [ ] You should move this to App class. See expanded reasoning in the comment below.